src/redux/selectedChargingStationSlice.tsx

```diff
@@ -1,12 +1,15 @@
 import { createSelector, createSlice, PayloadAction } from '@reduxjs/toolkit';
+
 import { RootState } from './store';
 import { ChargingStation } from '../pages/charging-stations/ChargingStation';
 
 interface SelectedChargingStationState {
+  chargingStations: ChargingStation[];
```

Comment:
what is the difference between `chargingStations` and `selectedChargingStation` here? I see below:
```
export const getSelectedChargingStation = () =>
  createSelector(
    (state: RootState) =>
      state.selectedChargingStationSlice.selectedChargingStation,
    (selectedChargingStation: any) =>
      selectedChargingStation ? JSON.parse(selectedChargingStation) : undefined,
  );

// Selector to get the full list of charging stations
export const getChargingStations = createSelector(
  (state: RootState) => state.selectedChargingStationSlice.chargingStations,
  (chargingStations: ChargingStation[]) => chargingStations,
);
```
both seem to be for "selected charging station(s)", can we clean this up to just have one variable for `selectedChargingStations` which is a list? that way we can also clean up the `setSelectedChargingStation` and maybe just have `addSelectedChargingStation` and `removeSelectedChargingStation` so that it can initialize with empty list and there can be actions to add/remove items from the list? 🙏

Reply:
Valid point. Did not want to break existing functionality when I was implementing it and I forgot to clean up

Reply:
It may also be helpful if we save it as a map where the key is the station ID and value is full object

Reply:
Implemented to use a map going forward which gives us a O(n)